File: src/net/eduard/api/command/api/ApiReloadCommand.java

package net.eduard.api.command.api;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.plugin.Plugin;

import net.eduard.api.lib.Mine;
import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.server.EduardPlugin;

public class ApiReloadCommand extends CommandManager {

	public ApiReloadCommand() {
		super("reload", "recarregar");
		setUsage("/api reload <plugin>");
		setDescription("Recarrega um plugin feito pelo Eduard");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		if (args.length == 1) {
			sendUsage(sender );
		} else {
			String pluginName = args[1];
			if (Mine.existsPlugin(sender, pluginName)) {
				Plugin plugin = Mine.getPlugin(pluginName);
				if (plugin instanceof EduardPlugin) {
					EduardPlugin eduardPlugin = (EduardPlugin) plugin;
					eduardPlugin.reload();
					sender.sendMessage("§bEduardAPI §aPlugin do Eduard §2"+plugin.getName()+"§a foi recarregado");
					
				}else {
					sender.sendMessage("§cEste plugin nao é um plugin do Eduard");
				}
			}
		}

		return true;
	}

}

File: src/main/java/net/eduard/api/command/api/ApiReloadCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.Mine
import net.eduard.api.server.EduardPlugin
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class ApiReloadCommand : CommandManager("reload", "recarregar") {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (args.isEmpty()) {
            sendUsage(sender)
            return true
        }
        val pluginName = args[0]
        if (Mine.existsPlugin(sender, pluginName)) {
            val plugin = Mine.getPlugin(pluginName)
            if (plugin is EduardPlugin) {
                plugin.reload()
                sender.sendMessage("§bEduardAPI §aPlugin do Eduard §f" + plugin.getName() + " §afoi recarregado")
            } else {
                sender.sendMessage("§cEste plugin nao é um plugin do Eduard")
            }
        }

        return true
    }

    init {
        usage = "/api reload <plugin>"
        description = "Recarrega um plugin feito pelo Eduard"
    }
}